lib/feature/home/presentation/widgets/photo_tab/button_tabs.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:file_share/core/presentation/palette.dart';
+import 'package:flutter/material.dart';
+
+class ButtonTabs extends StatefulWidget {
+  final int photoCount;
+  const ButtonTabs({
+    super.key,
+    required this.photoCount,
+  });
+
+  @override
+  State<ButtonTabs> createState() => _ButtonTabsState();
+}
+
+class _ButtonTabsState extends State<ButtonTabs> {
+  bool imagesButton = true;
+  bool albumsButton = false;
+  int selectedButtonIndex = 0;
+
+  void toggle(int index) {
+    setState(() {
+      selectedButtonIndex = index;
+    });
+    if (selectedButtonIndex == 0) {
+      imagesButton = true;
+      albumsButton = false;
+    } else {
+      albumsButton = true;
+      imagesButton = false;
+    }
+    setState(() {});
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        GestureDetector(
```

Comment:
These "buttons" are very similar. You can create one reusable widget here.
Instead of `imagesButton` and `albumsButton`, why not track just one state `selectedButtonIndex `?
Then for the color of the first button, it'll be something like:

```dart
color: selectedButtonIndex == 0 ? Palette.blue : Colors.grey[300],
```